core/debug/elf/elf.odin

```diff
@@ -0,0 +1,420 @@
+package elf
+
+/*
+Handy References:
+- https://refspecs.linuxbase.org/elf/elf.pdf
+- http://man7.org/linux/man-pages/man5/elf.5.html
+*/
+
+ELFCLASS32  :: 1
+ELFCLASS64  :: 2
+ELFDATA2LSB :: 1
+ELFDATA2MSB :: 2
+
+File_Type :: enum {
+	none        = 0x0,
+	relocatable = 0x1,
+	executable  = 0x2,
+	shared_obj  = 0x3,
+	core        = 0x4,
+	lo_os       = 0xFE00,
+	hi_os       = 0xFEFF,
+	lo_proc     = 0xFF00,
+	hi_proc     = 0xFFFF,
```

Comment:
The Odin convention is to have `enum` constants using `Ada_Case`. This however doesn't seem to be matching either the Odin convention, nor the convention used in the original ELF documentation.